cypress/component/DataSubmissionStudyInformation/ds_study_information.spec.js

```diff
@@ -0,0 +1,302 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { mount } from 'cypress/react';
+import { cloneDeep } from 'lodash/fp';
+import { User } from '../../../src/libs/ajax';
+import DataSubmissionStudyInformation from '../../../src/components/data_submission/ds_study_information';
```

Comment:
maybe the overall folder could be called `DataSubmission` so all the component tests for the overall functionality are in one place ?

Reply:
I'm sorry, I don't understand your suggestion - What do you mean?